### PR TITLE
A ifunctionality

### PR DESCRIPTION
**Pull request by taiwoAkindele**
changes discussed in the standup made (spacing between the education boxes and the title, footer links opening on new page)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -38,25 +38,23 @@ const App:React.FC = () => {
   return (
     <ThemeProvider theme={theme}>
       <div>
```
Comment (pablo-clueless) [inline src/App.tsx:41]: Why are you overriding the `Suspense` component?

File: src/sections/Hero.tsx [hunk 2]
```diff
@@ -13,18 +13,17 @@ const Hero: React.FC = () => {
   const handleSubmit = async(e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault()
     try {  
-      const json = JSON.parse(query);
```
Comment (pablo-clueless) [inline src/sections/Hero.tsx:16]: Why are you requiring the user to enter a JSON string?? Why not simplify it and let them enter plain text??

Comment (taiwoAkindele) [reply src/sections/Hero.tsx:16]: That was the request from @PhilemonBrain.

Comment (pablo-clueless) [reply src/sections/Hero.tsx:16]: Ok.

**Review by pablo-clueless — APPROVED** [review]

**Pull request merged**